src/group-configurations/GroupConfigurations.scss

```diff
@@ -23,8 +23,7 @@
   .configuration-card-header {
     display: flex;
     align-items: center;
-    align-content: center;
-    justify-content: space-between;
+    place-content: center space-between;
 
     .configuration-card-header__button {
       display: flex;
```

Comment:
Cool.. did I do that right?

Reply:
Nope, I meat you can use those classes instead of using a custom class to apply the same styles. 

i.e. do a search-replace with:
search `className="configuration-card-header"`
replace `className="configuration-card-header d-flex align-items-start p-0 h-auto text-black "`

Then these can be removed from the custom css

Reply:
Ugh.. I can do that, but `className="configuration-card-header"` is used in [4 separate places in the code](https://github.com/search?q=repo%3Aopenedx%2Ffrontend-app-course-authoring+%22configuration-card-header%22&type=code). Shouldn't we avoid putting layout classes in the HTML?

Reply:
@xitij2000 My attempt to extend those classes [broke the build](https://github.com/openedx/frontend-app-course-authoring/actions/runs/9494632406/job/26165346510?pr=1052).. is it ok if I just keep my original changes here?

Reply:
Let's try to keep this focused on the minimal changes we need to get these version bumps done. Our team (and now apparently TNL as well) has lots of PRs building up that are waiting for this one to get merged. So even though such cleanups would be nice, let's leave them for another time unless they're required.